Sources/FormattingHelpers.swift

```diff
@@ -1905,19 +1927,21 @@ extension Formatter {
             }
         }
 
-        func shouldBeMarked(in declarationTypes: [DeclarationType]) -> Bool {
+        func shouldBeMarked(in _: [DeclarationType]) -> Bool {
```

Comment:
If this argument is unused let's clean it up

```suggestion
        func shouldBeMarked() -> Bool {
```